### DISCUSSION TITLE
Approval Timeline Requirements

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#248] [General]
Can we make sure the system is aware that we need a minimum of 2 months from application to approval?

**Comment by LarsKemmann** [comment]
This is really interesting. Is that just a policy, basically that you don't approve volunteers until at least two months have passed? I'm curious what drives that and if there are any other timeline-related requirements that might exist.

For now, the manual workaround would be to have the 'director's approval' step wait on that.

**Reply by AGNMM18** [reply]
It's a requirement from our insurance.  That's the only timeline requirement I'm aware of.  OK, so this is just something to remember when approving volunteers (by the director), is that correct?

**Comment by LarsKemmann** [comment]
For now, yes.
